samples/sensor/stm32_vbat_sensor/README.rst

```diff
@@ -7,7 +7,7 @@ Overview
 ********
 
 This sample reads the Vbat from the STM32 Internal
```

Comment:
we need to rename the sample to not be stm32 specific, this is now a generic sample that can work on any board, and can measure any voltage, not just battery.